### PR TITLE
Scaffold bubble chart

### PR DESCRIPTION
**Pull request by aabdaab**
This PR creates a **scaffold** layout for a bubble chart with **mock data** and the grid:

image

The chart is created with the help of `victory` library.

Please, check `http://localhost:3000/tpi/sectors`

### CODE REVIEW

File: app/assets/stylesheets/tpi/_bubble-chart.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
+$tape-height: 8px;
+$tape-color: rgba(25,25,25,0.1);
+
+.bubble-chart__container {
```
Comment (rapzo) [inline app/assets/stylesheets/tpi/_bubble-chart.scss]: Sorry for nitpicking but can we use `--element__modifier`instead?

**Comment by aabdaab** [comment]
The next step is: pass real data, play with colors, polish the design, proper headers etc. Gonna do it in a different PR 👩‍🎨

**Comment by rapzo** [comment]
Great stuff!

File: app/assets/stylesheets/tpi/_bubble-chart.scss (hunk 1 above)
Comment (aabdaab) [reply app/assets/stylesheets/tpi/_bubble-chart.scss]: yes! sorry, my BEM is rusty

**Comment by tsubik** [comment]
Wow, Nice!! You are the best @aabdaab !

**Comment by simaob** [comment]
Great Success 👍

**Comment by aabdaab** [comment]
💪

File: app/javascript/components/BubbleChart.js [hunk 2]
```diff
@@ -1,13 +1,53 @@
...
+          </VictoryGroup>
+        </div>
+      ))}
+    </>
+  )
+}
+
+const BubbleChart = () => {
+  return (
+    <React.Fragment>
```
Comment (tsubik) [inline app/javascript/components/BubbleChart.js]: this react fragment is not needed here, right?

File: app/assets/stylesheets/tpi/_bubble-chart.scss (hunk 1 above)
Comment (aabdaab) [reply app/assets/stylesheets/tpi/_bubble-chart.scss]: done

File: app/javascript/components/BubbleChart.js (hunk 2 above)
Comment (aabdaab) [reply app/javascript/components/BubbleChart.js]: true - removed it

**Pull request merged**